client/state/ui/olark/selectors.js

```diff
@@ -35,6 +35,16 @@ export function isRequestingOlark( state ) {
 }
 
 /**
+ * Returns if olark chat is available for the given area
+ * @param   {Object}  state  Global state tree
+ * @param   {object}  area   The chat area to check availability for
+ * @returns {Boolean}        true, when olark is requesting
+ */
+export function isChatAvailable( state, area ) {
```

Comment:
Wonder if a variable named uiContext or something similar might be a hair more descriptive?